Remove bogus file: prefix when saving the index file,
remove useless overwrite confirmation.

--- a/src/dvbcut.cpp~	2010-08-18 12:10:59.531214776 +0200
+++ b/src/dvbcut.cpp	2010-08-18 12:20:31.222922627 +0200
@@ -1869,13 +1869,14 @@
 	  u.setProtocol(QString("file"));
 	  u.setPath(QString::fromStdString(idxfilename));
 	  if (chdir((const char*)u.dirPath()) == -1) chdir("/");
-	  QString relname = QString("file:") + u.fileName();
+	  QString relname = u.fileName();
 	  QString s=QFileDialog::getSaveFileName(
-		  relname,
-		  settings().idxfilter,
 		  this,
-		  "Choose index file...",
-		  "Choose the name of the index file" );
+		  "Choose the name of the index file",
+		  relname,
+		  settings().idxfilter,
+		  0,
+		  QFileDialog::DontConfirmOverwrite);
 	  if (s.isEmpty()) {
 		delete mpg;
 		mpg=0;

